src/style.css

```
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.big__form {
+  position: relative;
+  box-sizing: border-box;
+  width: calc(100% - 50px);
+  height: 70px;
+  left: 50px;
+  top: 20px;
```

pay attention to readme:
- search bar must have width 100%
- distance between two search bars must be 20px
- the big search bar must have top indent 20px
```suggestion
  height: 70px;
```